extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -736,13 +736,13 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       'data-multi-size-validation': true,
       'data-override-width': true,
       'data-override-height': true,
-      'data-json': true,
+      'json': true,
```

Comment:
Let's remove json from the whitelist otherwise vendors will inadvertently send the entire json attribute value via ATTR(json)

Reply:
done